functions/resources/public/css/_reset.scss

```diff
@@ -0,0 +1,59 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
Ouch. We actually need a full reset file?

I'm not very good with CSS. What all does this do for us? Use a framework piecemeal or something?

Reply:
This is mostly standard boilerplate doing a few things to keep our sanity:

1) Stripping out cross browser inconsistencies in rendering font, tags and forms

2) Forcing everything into box-sizing (believe me, you want this) see https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

3) Forcing full-size for base elements to ease layout by default